### PR TITLE
Remove Step1 once agreement has been accepted.

### PR DESCRIPTION
**Pull request by MayankR**
Solution for the bug https://github.com/mozilla/addons-server/issues/1696 .

### CODE REVIEW

**Comment by andymckay** [comment]
It looks like there's some failing unit tests which rely on their being a specific number of steps and those will need updating.

File: src/olympia/devhub/templates/devhub/addons/submit/sidebar.html [hunk 1]
```diff
@@ -19,24 +19,27 @@
...
-          {% else %}
-            <a href="{{ url(BASE % loop.index, addon.slug) }}">{{ text }}</a>
-          {% endif %}
-        {% endif %}
-      </li>
+      {# Skip adding the first list item (Getting Started) if already accepted. #}
+      {% if step.current == 1 or loop.index > 1 %}
+         <li class="{{ class }}{% if step.current == loop.index %} current{% endif %}">
+           {% if step.current < HAS_ADDON %}
+             {% if loop.index <= step.current %}
```
Comment (mstriemer) [inline src/olympia/devhub/templates/devhub/addons/submit/sidebar.html]: I think you want to update this `if` instead so the text is there but it isn't a link.

**Comment by MayankR** [comment]
I changed it so that only Step 1 would simply be text and not link once we move to Step 2. However the build is still failing as I think it is checking if the two steps are still linked in `self.assert_linked(doc, [1, 2])`.

**Comment by mstriemer** [comment]
I think you change that line to `self.assert_linked(doc, [2])`. It's in [src/olympia/devhub/tests/test_views.py on line 2020](https://github.com/mozilla/addons-server/blob/6fe5a1d3d9ccba5a0fe4224f4d2be69f9d460a29/src/olympia/devhub/tests/test_views.py#L2020).

Can you please squash your changes down to one commit when you're done?

**Comment by andymckay** [comment]
It looks like the failed test is not related to the change @mstriemer ?

**Comment by EnTeQuAk** [comment]
This looks good and tests pass now, let's merge this @mstriemer?

**Comment by mstriemer** [comment]
Thanks for the fix!

**Pull request closed without merging**